awspub/image.py

```diff
@@ -556,6 +557,14 @@ def publish(self) -> None:
                     f"currently using partition {partition}. Ignoring marketplace config."
                 )
 
+        # send ssn notification
+        if self.conf["sns"]:
```

Comment:
we need to have the possibility in the future to send SNS message only if images are public. likely this needs to be configurable per topic.

Reply:
SNS notification is configured in each image. Private images also can send notification with allowed-only subscribed topics.